bitnami/scylladb/values.yaml

```diff
@@ -0,0 +1,1115 @@
+##
+schedulerName: ""
+## @param terminationGracePeriodSeconds In seconds, time the given to the Scylladb pod needs to terminate gracefully
+## ScyllaDB requires more time to have the node drained that's why we're setting a default value
+## ref: https://kubernetes.io/docs/concepts/workloads/pods/pod/#termination-of-pods
+##
+terminationGracePeriodSeconds: ""
+## @param extraVolumes Optionally specify extra list of additional volumes for scylladb container
+##
+extraVolumes: []
+## @param extraVolumeMounts Optionally specify extra list of additional volumeMounts for scylladb container
+##
+extraVolumeMounts: []
+## @param initContainers Add additional init containers to the scylladb pods
+##
+initContainers: []
+## @param sidecars Add additional sidecar containers to the scylladb pods
+##
+sidecars: []
+## Scylladb Pod Disruption Budget configuration
+## ref: https://kubernetes.io/docs/tasks/run-application/configure-pdb/
+##
+pdb:
+  ## @param pdb.create Enable/disable a Pod Disruption Budget creation
+  ##
+  create: true
+  ## @param pdb.minAvailable Mininimum number of pods that must still be available after the eviction
+  ##
+  minAvailable: 1
+  ## @param pdb.maxUnavailable Max number of pods that can be unavailable after the eviction
+  ##
+  maxUnavailable: ""
+## @param hostNetwork Enable HOST Network
+## If hostNetwork true ->  dnsPolicy is set to ClusterFirstWithHostNet
+##
+hostNetwork: false
+## Scylladb container ports to open
+## If hostNetwork true: the hostPort is set identical to the containerPort
+## @param containerPorts.intra Intra Port on the Host and Container
+## @param containerPorts.tls TLS Port on the Host and Container
+## @param containerPorts.jmx JMX Port on the Host and Container
+## @param containerPorts.cql CQL Port on the Host and Container
+## @param containerPorts.cqlShard CQL Port (Shard) on the Host and Container
+## @param containerPorts.api REST API port on the Host and Container
+## @param containerPorts.metrics Metrics port on the Host and Container
+##
+containerPorts:
+  intra: 7000
+  tls: 7001
+  jmx: 7199
+  cql: 9042
+  cqlShard: 19042
+  api: 10000
+  metrics: 9180
+## @param extraContainerPorts Optionally specify extra list of additional ports for the container
+## e.g:
+## extraContainerPorts:
+##   - name: myservice
+##     containerPort: 9090
+##
+extraContainerPorts: []
+## Scylladb ports to be exposed as hostPort
+## If hostNetwork is false, only the ports specified here will be exposed (or not if set to an empty string)
+## @param hostPorts.intra Intra Port on the Host
+## @param hostPorts.tls TLS Port on the Host
+## @param hostPorts.jmx JMX Port on the Host
+## @param hostPorts.cql CQL Port on the Host
+## @param hostPorts.cqlShard CQL (Sharded) Port on the Host
+## @param hostPorts.api REST API Port on the Host
+## @param hostPorts.metrics Metrics Port on the Host
+##
+hostPorts:
+  intra: ""
+  tls: ""
+  jmx: ""
+  cql: ""
+  cqlShard: ""
+  api: ""
+  metrics: ""
```

Comment:
Are we adding support for this for any particular reason?

Reply:
This was supported in bitnami/cassandra and we want this chart to have a similar UX